plutus-conformance/README.md

```diff
@@ -15,6 +15,10 @@ The tests currently covers or will cover:
 - Costing conformance 
 - Coverage test
 
+There is also an executable `add-test-output` for easy addition of test cases. Run 
```

Comment:
I wonder if we could simplify things by using `tasty-golden`, like we do in the rest of the codebase. Then you can create the expected files by just passing `--test-options=--accept` when running the test suite.

Reply:
I'd be fine with any solution as long as it allows us to spit out automatically generated files thus emulating property tests. I think `tasty-golden` is designed with a different paradigm in mind, so a separate executable seems like a better solution, especially given the fact that we can customize it any way we want, unlike `tasty-golden`.